src/OrleansServiceBus/Providers/Streams/EventHub/IEventHubSettings.cs

```diff
@@ -8,6 +8,13 @@ public interface IEventHubSettings
         string ConsumerGroup { get; }
         string Path { get; }
         int? PrefetchCount { get; }
+
+        /// <summary>
+        /// Indicates if stream provider should read all new data in partition, or from start of partition.
+        /// True - read all new data added to partition.
+        /// False - start reading from beginning of partition.
+        /// Note: If checkpoints are used, stream provider will always begin reading from most recent checkpoint.
+        /// </summary>
         bool StartFromNow { get; }
```

Comment:
If it matters, for an "untrained eye" it would look more clear with an enum. Maybe something like `ReadFromBeginning`, `ReadFromCheckpoint` (or what could be the word what's new data only).